browser/src/control/Control.NotebookbarWriter.js

```diff
@@ -1085,6 +1085,12 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 				],
 				'vertical': 'true'
 			},
+			{
+				'id':'toggledarktheme',
+				'type': 'bigmenubartoolitem',
```

Comment:
Cool, nice to use bigmenubartoolitem so we can share stugg between Menubar and notebookbar :+1: